app/mixins/setup-submit-controller.js

```diff
@@ -21,7 +22,11 @@ export default Ember.Mixin.create({
                 controller.set('providers', providers);
             }
         );
-
+        this.get('theme.provider').then(provider => {
+            provider.query('licensesAcceptable', {'page[size]': 20}).then(licenses => {
```

Comment:
The spec says `The default license will be chosen by the service (OSFPreprints will start with CC0)`. My default license for the OSF is showing up as "Academic Free License (AFL) 3.0", the first alphabetically. But maybe the data isn't properly populated yet.